src/vorta/views/archive_tab.py

```diff
@@ -330,12 +332,14 @@ def on_selection_change(self, selected=None, deselected=None):
         if not self.repoactions_enabled:
             reason = self.tr("(borg already running)")
 
-        # toggle delete button
+        # Disable the delete and refresh buttons if no archive is selected
         if self.repoactions_enabled and len(indexes) > 0:
             self.bDelete.setEnabled(True)
+            self.bRefreshArchive.setEnabled(True)
             self.bDelete.setToolTip(self.tooltip_dict.get(self.bDelete, ""))
         else:
             self.bDelete.setEnabled(False)
+            self.bRefreshArchive.setEnabled(False)
```

Comment:
According to the HIG disabled widgets should state a reason for being disabled when they are by enclosing the reason with brackets and appending it to the tooltip. The refresh button is missing that.